.gitignore

```diff
@@ -2,4 +2,5 @@
 *.pyc
 .DS_Store
 .directory
-/node_modules
\ No newline at end of file
+/node_modules
+resources/todomvc/architecture-examples/svelte/node_modules
```

Comment:
Should we just ignore all node_modules folders instead of one-off. I believe the plan for this is that we check in all package lock files so that we don't need to bloat the repo with the actual modules (relying on build tools to make sure we have a page that can be run from a static server)